code/weapon/weapons.cpp

```diff
@@ -2344,12 +2344,24 @@ int parse_weapon(int subtype, bool replace, const char *filename)
 			wip->cmeasure_firewait = (int) (wip->fire_wait*1000.0f);
 		}
 
-		if (optional_string("+Fail Delay Equals Fire Wait:")) {
-			stuff_boolean(&wip->cmeasure_faildelay_equals_firewait);
+		if (optional_string("+Failure Launch Delay Multiplier for AI:")) {
+			int delay_multiplier;
+			stuff_int(&delay_multiplier);
+			if (delay_multiplier > 0) {
+				wip->cmeasure_failure_delay_multiplier_ai = delay_multiplier;
+			} else {
+				mprintf(("Warning: \"+Failure Launch Delay Multiplier for AI:\" should be >= 0 (read %i). Value will not be used. ", delay_multiplier));
```

Comment:
Should be a legit `Warning()`, this will merely silently print to the debug log, look around you'll see how others do it in this area. Ditto below.

Reply:
Good idea, just updated.